twisted-client-2/get-poetry-simple.py

```diff
@@ -73,7 +73,10 @@ def __init__(self, poetry_count):
 
     def poem_finished(self, poem=None):
         if poem is not None:
-            self.poems.append(poem)
+            # this seems like the place to convert poem from bytes to unicode
+            # the network code dealt with bytes
+            # but the completed poem should be a unicode string.
```

Comment:
I don't think this comment is needed, I agree with putting the conversion here.

Reply:
removed and pushed.